cypress/page/shoping-cart.page.ts

```diff
@@ -0,0 +1,21 @@
+class ShopingCartPage {
+    private ShopingCart: string;
+    private CartNavigation: string
+
+    constructor() {
+        this.ShopingCart = '[style*="display: block;"] .button-container > a';
```

Comment:
since in the `products-list.page.ts` file we are already adding the T-Shirt to the cart, this field is not needed.